test.sh

```diff
@@ -7,23 +8,29 @@ PACKAGES=$(go list ./... | grep -v /vendor/)
 
 go vet $PACKAGES
 if ! which fgt > /dev/null ; then
-  echo "Please install fgt from https://github.com/GeertJohan/fgt."
-  exit 1
+    echo "Please install fgt from https://github.com/GeertJohan/fgt."
+    exit 1
 fi
 
 if ! which golint > /dev/null ; then
-  echo "Please install golint from github.com/golang/lint/golint."
-  exit 1
+    echo "Please install golint from github.com/golang/lint/golint."
+    exit 1
 fi
 
 for package in $PACKAGES
 do
-        fgt golint ${package##github.com/cloudflare/cfssl/}/*.go
+    fgt golint ${package##github.com/cloudflare/cfssl/}/*.go
 done
 
 # Build and install cfssl executable in PATH
 go install github.com/cloudflare/cfssl/cmd/cfssl
-
 go test $BUILD_FLAGS $PACKAGES
```

Comment:
Test duration doubled with this commit: ![](http://i.imgur.com/qAi3IUq.png)

https://github.com/andrewbuss/cfssl/commit/27edf77db2d43126cecdeae13a56684b7d0623ff#diff-1b531c5e405e0bc95890d52cf2f2f5b5L27 unduplicates the test